src/Symfony/Component/Config/Resource/ClassExistenceResource.php

```diff
@@ -43,15 +43,15 @@ public function __construct(string $resource, bool $exists = null)
     /**
      * {@inheritdoc}
      */
-    public function __toString()
+    public function __toString(): string
```

Comment:
It seems the return type was not added consistently in the Config/Resource/ classes. It's added here for example, but not in ComposerResource::__toString, or ComposerResource::getVendors, or GlobResource::getIterator, or GlobResource::getPrefix etc.
It currently seems arbitrary where return types have been added. Was this a manual process?

Reply:
It was mostly a manual process yes.
Lot of methods could have `void` returned in some cases, e.g., from your examples,  GlobResource::getIterator

I rechecked the classes, and added missing types.

Reply:
GlobResource::getIterator should not return void but `\Traversable`